_aggregations/metric/matrix-stats.md

```diff
@@ -9,8 +9,23 @@ redirect_from:
 
 # Matrix stats aggregations
 
-The `matrix_stats` aggregation generates advanced stats for multiple fields in a matrix form.
-The following example returns advanced stats in a matrix form for the `taxful_total_price` and `products.base_price` fields:
+The `matrix_stats` metric is a multi-value metric that generates covariance statistics for two or more fields in matrix form. 
+
+The `matrix_stats` metric does not support scripting.
+{: .note}
+
+## Parameters
+
+The `matrix_stats` aggregation takes the following parameters: 
+
+| Parameter | Required/Optional | Data type      | Description |
```

Comment:
I tested the `missing` parameter and it gave me an error. Does `missing` not work with `matrix-stats`?

Reply:
@jainankitk Could you please answer this quiestion?

Reply:
@jainankitk Maybe `missing` doesn't work because it takes an array of input fields? I tried the following and got an error:
```
GET opensearch_dashboards_sample_data_ecommerce/_search
{
  "size": 0,
  "aggs": {
    "matrix_stats_taxful_total_price": {
      "matrix_stats": {
        "fields": ["taxful_total_price", "products.base_price"],
        "missing": [0, 0]
      }
    }
  }
}
```

Reply:
Okay you need to provide map of fieldname:value in the missing parameter, like below:

```
{
  "size": 0,     
  "aggs": {
    "matrix_stats_taxful_total_price": {
      "matrix_stats": {
        "fields": ["status", "size"],
        "missing": {"status":0}
}}}}
```